js_modules/dagster-ui/packages/ui-core/src/app/UserSettingsButton.tsx

```diff
@@ -34,11 +44,51 @@ const SettingsButton = styled.button`
 
 export const UserSettingsButton = () => {
   const [isOpen, setIsOpen] = React.useState(false);
+  const [shouldHideUIThemeNux, setShouldHideUIThemeNux] = useStateWithStorage<string | null>(
+    'dagster-theme-nux',
+    (value) => value,
+  );
+
+  const button = (
+    <SettingsButton onClick={() => setIsOpen(true)} title="User settings">
+      <Icon name="settings" color={CoreColors.Gray200} />
+    </SettingsButton>
+  );
+
+  const target = shouldHideUIThemeNux ? (
+    button
+  ) : (
+    <ProductTour
+      title="New: UI Themes"
+      description={
+        <span>You can now select a dark or light mode UI theme from your user settings.</span>
+      }
+      position={ProductTourPosition.BOTTOM_RIGHT}
+      object={
+        <div style={{borderRadius: 6, overflow: 'hidden'}}>
+          <Image
+            src={uiThemes}
+            alt="UI Themes"
+            width={230}
+            height={130}
+            style={{display: 'block'}}
+          />
+        </div>
+      }
+      actions={{
+        dismiss: () => {
+          setShouldHideUIThemeNux('1');
+        },
+      }}
+      modifiers={{offset: {enabled: true, options: {offset: [-12, 8]}}}}
+    >
+      {button}
+    </ProductTour>
```

Comment:
Wow I think I'd missed this in other PRs, this is a really nice component interface 💯